src/libguac/Makefile.am

```diff
@@ -100,8 +100,8 @@ libguac_la_LDFLAGS =    \
     @PNG_LIBS@          \
     @PTHREAD_LIBS@      \
     @UUID_LIBS@         \
-    @VORBIS_LIBS@
+    @VORBIS_LIBS@	\
+    @JPEG_LIBS@
```

Comment:
Mind moving `@JPEG_LIBS@` up, between `@CAIRO_LIBS@` and `@PNG_LIBS@`, such that the existing alphabetical ordering is maintained?